python/rikai/types/vision.py

```diff
@@ -160,7 +160,11 @@ def display(self, **kwargs):
             return Image(fobj.read(), **kwargs)
 
     def __repr__(self) -> str:
-        return f"Image(uri={self.uri})"
+        return (
```

Comment:
Nit: it's probably more readable as just a regular if/else instead of doing it inline